content/_partials/community-role-holder.html.haml

```diff
@@ -0,0 +1,26 @@
+- authorId = page.author || ''
+- author = site.authors[authorId]
+- if author
+  %div{:id => "about-the-author"}
+    - unless page.blogroll.nil?
+      %h2 #{author.name}
+    %table.author.box
+      %tr
+        %td
+          = partial("avatar.html.haml", :author => author, :authorId => authorId)
+        %td
+          - if page.blogroll.nil?
```

Comment:
Is the idea to merge this with /blog/author/xxx? If no, it can be simplified (e.g. remove all references to blogroll)

Reply:
I would rather merge it later. A bit of copy-pasta